loadtest/config.json

```diff
@@ -60,6 +72,7 @@
         "payload": "{}"
       }
     },
+    "sei_tester_address": "",
```

Comment:
what is this for?

Reply:
thats the address of the deployed contract `sei-tester` that's used for the parallelism transactions + iterator workloads